.github/deploy.yml

```diff
@@ -0,0 +1,37 @@
+name: Build / Test / Deploy
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - develop
+      - main
```

Comment:
let's disable this on push for now (comment this out) - until we validate via manual invocations it is doing what we expect

Reply:
removed